src/Microsoft.AspNet.Razor.Runtime/TagHelpers/TagHelperContent.cs

```diff
@@ -13,7 +11,7 @@ namespace Microsoft.AspNet.Razor.Runtime.TagHelpers
     /// <summary>
     /// Abstract class used to buffer content returned by <see cref="ITagHelper"/>s.
     /// </summary>
-    public abstract class TagHelperContent : IEnumerable<string>, IHtmlContent
+    public abstract class TagHelperContent : IHtmlContent
```

Comment:
I talked to @dougbu and @rynowak and where I stand now is that this indeed shouldn't implement `IEnumerable<anything>`.